src/paulscode/android/mupen64plusae/GalleryActivity.java

```diff
@@ -220,6 +271,49 @@ public void onClick( MenuItem menuItem )
             CharSequence message = getText( R.string.invalidInstall_message );
             new Builder( this ).setTitle( title ).setMessage( message ).create().show();
         }
+        
+        if( savedInstanceState != null )
+        {
+            mSelectedItem = null;
+            String md5 = savedInstanceState.getString( STATE_SIDEBAR );
+            if( md5 != null )
+            {
+                // Repopulate the game sidebar
```

Comment:
Without this, opening the game sidebar then rotating your device will show the navigation drawer, as that's the default view when creating the activity.